### PR TITLE
marshmallow: Include all links which are not hidden to fulltext_links

### PR DESCRIPTION
**Pull request by pazembrz**
*INSPIR-3268

### CODE REVIEW

File: backend/inspirehep/records/marshmallow/literature/ui.py [hunk 1]
```diff
@@ -129,11 +129,12 @@ def get_arxiv_fulltext_link(self, data):
         return missing
 
     def get_internal_fulltext_link(self, data):
+        # Assume all non hidden are fulltext - INSPIR-3268
```
Comment (monaawi) [inline backend/inspirehep/records/marshmallow/literature/ui.py]: I don't think this is needed as we can trace back the change to the commit and you have the JIRA task in the commit message

File: backend/tests/unit/records/marshmallow/literature/test_ui.py [hunk 2]
```diff
@@ -76,8 +76,16 @@ def test_internal_fulltext_hidden(current_app_mock):
 
 
 @mock.patch("inspirehep.records.marshmallow.literature.ui.current_app")
-def test_internal_not_fulltext(current_app_mock):
+def test_not_marked_as_fulltext_in_fulltext(current_app_mock):
```
Comment (monaawi) [inline backend/tests/unit/records/marshmallow/literature/test_ui.py]: maybe ```test_non_fulltext_documents_appear_in_fulltext_links_field```?

File: backend/inspirehep/records/marshmallow/literature/ui.py (hunk 1 above)
Comment (pazembrz) [reply backend/inspirehep/records/marshmallow/literature/ui.py]: :+1:

File: backend/tests/unit/records/marshmallow/literature/test_ui.py (hunk 2 above)
Comment (pazembrz) [reply backend/tests/unit/records/marshmallow/literature/test_ui.py]: :+1:

**Pull request merged**